hax/hax/util.py

```diff
@@ -441,7 +448,11 @@ def update_process_status(self, event: ConfHaProcess) -> None:
         assert 0 <= event.chp_event < len(ha_process_events), \
             f'Invalid event type: {event.chp_event}'
 
-        data = json.dumps({'state': ha_process_events[event.chp_event]})
+        LOG.info('Updating process status type=%s event=%s',
+                 ha_process_types[event.chp_type],
+                 ha_process_events[event.chp_event])
+        data = json.dumps({'type': ha_process_types[event.chp_type],
```

Comment:
In many cases hax code uses simplejson.dumps/loads in favor of json.dumps/loads
As far as I remember this was caused by the way how poorly json module encodes/decodes Enums and our Fid. Please consider using `[dump_json](https://github.com/Seagate/cortx-hare/blob/main/hax/hax/util.py#L537)()` function from our util.py.

As a rule of a thumb, one probably should prefer simplejson instead of json module (until otherwise is proven).

Reply:
Okay, fixed.